packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcPreview.ts

```diff
@@ -145,11 +145,11 @@ export async function forceLightningLwcPreview(sourceUri: vscode.Uri) {
   const fullUrl = `${DEV_SERVER_PREVIEW_ROUTE}/${componentName}`;
   // Preform existing desktop behavior if mobile is not enabled.
```

Comment:
I know this was existing, but can we update this typo?
```suggestion
  // Perform existing desktop behavior if mobile is not enabled.
```